charmhelpers/contrib/openstack/utils.py

```diff
@@ -407,7 +411,7 @@ def get_os_version_codename(codename, version_map=OPENSTACK_CODENAMES):
             return k
     e = 'Could not derive OpenStack version for '\
         'codename: %s' % codename
-    error_out(e)
+    error_out(e, raiser=ValueError)
```

Comment:
This seems like a lot of work to go to?  couldn't you just do:

```python
raise ValueError(str(e))
```

instead if you wanted a change in behaviour here?

Reply:
I think it stemmed from an abundance of caution in context of the specific case I'm proposing this change for to not break other charms, but now that it has spilled over to the common codebase, this is indeed more sensible to do.